lua/conform/formatters/ktlint.lua

```diff
@@ -0,0 +1,10 @@
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://ktlint.github.io/",
+    description = "An anti-bikeshedding Kotlin linter with built-in formatter.",
+  },
+  command = "ktlint",
+  args = { "--format", "--stdin", "**/*.kt", "**/*.kts" },
```

Comment:
do you need the globs? Based on [the docs](https://pinterest.github.io/ktlint/1.0.1/install/cli/#stdin-stdout) it seems like those aren't necessary if you're passing `--stdin`

Reply:
Without paths it prints out
`18:40:35.484 [main] INFO com.pinterest.ktlint.cli.internal.KtlintCommandLine - Enable default patterns [**/*.kt, **/*.kts]`
1) I don't know how to exclude stderr from output
2) Those paths are enabled automatically

Any hints?

Reply:
I saw null-ls.nvim and they did the same

Reply:
From the same docs:
`Logging output printed to stdout can be suppressed by setting --log-level=none`

Not sure why that _wouldn't_ be enabled by default when using stdin/stdout, but it seems like a good place to start